### PR TITLE
Completed dom-querying exercise - Andrew Almeida

### PR DESCRIPTION
**Pull request by Andrew-M-A**
dom querying

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+console.log('hello, world!');
+
+console.log(document.querySelector('h1'));
+console.dir('h1');
```
Comment (codingcodymiller) [inline dom-querying/main.js:4]: This is not what the exercise requested. You should be passing the DOM element itself as an argument here, not the CSS selector for it.

File: dom-querying/main.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+console.log(document.querySelector('#explanation'));
+console.dir('#explanation');
```
Comment (codingcodymiller) [inline dom-querying/main.js:7]: This is not what the exercise requested. You should be passing the DOM element itself as an argument here, not the CSS selector for it.

File: dom-querying/main.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+console.log(document.querySelector('.hint'));
+console.dir('.class');
```
Comment (codingcodymiller) [inline dom-querying/main.js:10]: This is not what the exercise requested. You should be passing the DOM element itself as an argument here, not the CSS selector for it.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Andrew-M-A** [comment]
Screen Shot 2022-05-23 at 11 02 43 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**